src/pretix/base/models/event.py

```diff
@@ -550,16 +551,31 @@ def get_mail_backend(self, timeout=None, force_custom=False):
         Returns an email server connection, either by using the system-wide connection
         or by returning a custom one based on the event's settings.
         """
-        from pretix.base.email import CustomSMTPBackend
+        from pretix.base.email import CustomSMTPBackend, SendGridEmail
```

Comment:
Why in this PR, you put `import` lines to function body, instead of the top of the file?